packages/core/src/common/selection-command-handler.ts

```diff
@@ -33,7 +33,7 @@ export class SelectionCommandHandler<S> implements CommandHandler {
 
     isVisible(...args: any[]): boolean {
         const selection = this.getSelection(...args);
-        return !!selection && (!this.options.isVisible || (this.options.isVisible as any)(selection as any, ...args));
+        return !this.options.isVisible || (this.options.isVisible as any)(selection as any, ...args);
```

Comment:
I've pushed a different solution, with no breaking changes.

Reply:
I'm trying to think about this one a bit more, I'm not sure the proper or cleanest solution would be to force the `selection-command-handler` to suit our needs but maybe rather create a new `handler`, or even omit using the `selection-command-handler` and instead handle things ourselves with the `selectionService`.

Reply:
I agree this issue can be fixed in different ways. My basic understanding is that there is no reason why `SelectionCommandHandler` should force the visibility of item to always consider a selection though. I don't think it's completely unreasonable to let this be decided for every specific command handler getting registered.

In any case I'm happy to change this PR if you have a better suggestion.

Reply:
By definition and name the `SelectionCommandHandler` is based on selection so that makes sense. If we do not care about the selection like in this case then the handler shouldn't be used and we can instead handle it ourselves rather than try to fit it to our needs for this specific use-case.